nmaipy/parcels.py

```diff
@@ -116,13 +116,21 @@
     },
 }
 
-TREE_BUFFERS_M = dict(
+BUFFER_ZONES_M = dict(
```

Comment:
Just want to confirm that we will get buffer zones that are 0-5ft, 5-30ft, and 30-100ft from the building. Looks here like we're splitting the 5-30 ft into 5-10 and 10-30, which is fine but just extra postprocessing steps to get what we need for scores.

Reply:
See other comment - lets leave as independent "full buffers" here, and you can post calculate whatever rings you want (5-30, 10-30, etc) through subtraction.